### PR TITLE
enh(mp) Migrate Microsoft SCCM Monitoring Procedure to v2

### PR DESCRIPTION
**Pull request by mhuber-centreon**
Waiting for more info on the outputs

### CODE REVIEW

**Review by Sims24 — CHANGES_REQUESTED** [review]
Metrics name are wrong. 
Few indents issus

**Comment by thibaults-centreon** [comment]
A lot of fixes and changed have been made. @Sims24 please let me know if this sounds better.
Thanks

File: en/integrations/plugin-packs/procedures/applications-sccm-nsclient.md [hunk 1]
```diff
@@ -3,47 +3,207 @@ id: applications-sccm-nsclient
...
 Click on the *Save* button.
 
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the plugin installed, log into your Centreon Poller CLI using the *centreon-engine* user account and test the Plugin 
+by running the following command:
+
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-sccm-nsclient.md:142]: I can't find the closing docusaurus tab at the end.

File: en/integrations/plugin-packs/procedures/applications-sccm-nsclient.md [hunk 2]
```diff
@@ -3,47 +3,207 @@ id: applications-sccm-nsclient
...
+
+## Prerequisites 
+
+The Windows SCCM Plugin is hosted by the *centreon-nsclient* agent which must be installed, configured and running on the Windows server 
+hosting the SCCM feature.
+The Centreon Poller can connect to the agent using either the NRPE method or the RestAPI method.
+More information on how to achieve the installation and the configuration of the agent can be found in the associated procedure:
+
+* [NRPE](https://docs.centreon.com/current/en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.html)
+* [RestAPI](https://docs.centreon.com/current/en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-restapi.html)
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-sccm-nsclient.md:51]: Could you use relative paths instead ? ../../

**Review by Sims24 — APPROVED** [review]

**Review by Sims24 — APPROVED** [review]

**Pull request merged**